docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -70,18 +70,21 @@ This is an example of a minimal configuration that touches the values that most
 [Global]
 username=user  
 password=pass  
-auth-url=https://<openstack_endpoint>:5000/v3  
+auth-url=https://<keystone_ip>/identity/v3  
```

Comment:
I'm not sure this is correct. Maybe `<keystone_ip>:<keystone_port>/v3` would be better?

Reply:
this version is correct. the port based framework is fading away.